src/types/global.d.ts

```diff
@@ -20,7 +20,7 @@ export type Theme = {
       };
     };
   };
-  maxWidths: string[];
+  maxWidths: Array<string[]>;
```

Comment:
Doesn't this commit render type errors until d7db82aaf384dd625527cf7af42a96c7490aa827 is applied?